<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="WidgetCreateAccountWithInvalidCredentials" failures="1" tests="1" time="168.359" errors="0">
  <properties/>
  <testcase name="test_WidgetCreateAccountWithInvalidEmail" time="168.359" classname="com.sharethis.QAAutomation.createAccountUsingWidget.WidgetCreateAccountWithInvalidCredentials">
    <failure type="junit.framework.AssertionFailedError" message="expected:&lt;1&gt; but was:&lt;0&gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<1> but was:<0>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:277)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:195)
	at junit.framework.Assert.assertEquals(Assert.java:201)
	at com.sharethis.QAAutomation.commonlib.STCommonLibrary.stTearDown(STCommonLibrary.java:64)
	at com.sharethis.QAAutomation.createAccountUsingWidget.WidgetCreateAccountWithInvalidCredentials.test_WidgetCreateAccountWithInvalidEmail(WidgetCreateAccountWithInvalidCredentials.java:364)
... Removed 22 stack frames]]>
    </failure>
  </testcase>
</testsuite>
